frodo-runtime/src/main/java/com/fernandocejas/frodo/aspect/LogSubscriber.java

```diff
@@ -78,9 +80,14 @@ public void beforeOnStartExecution(JoinPoint joinPoint) {
   public void beforeOnNextExecution(JoinPoint joinPoint) {
     countAndMeasureTime();
     final FrodoJoinPoint frodoJoinPoint = new FrodoJoinPoint(joinPoint);
-    final Object value = frodoJoinPoint.getMethodParamValuesList().get(0);
-    messageManager.printSubscriberOnNext(joinPoint.getTarget().getClass().getSimpleName(),
-        value.toString(),
+    final String message;
```

Comment:
I haven't seen `MessageManager.class`.  Yes because we are encapsulating the handling on `printSubscriberOnNext`